docs/wxm-devices/helium-bundle/helium-troubleshooting.mdx

```diff
@@ -14,15 +14,7 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
 
 ### The updating process lasts forever
 
-<div class="explanation small-big">
-  <img src={useBaseUrl("img/wxm-devices/helium-bundle/reset_station.svg")} />
-  <span>
-
-If the process hangs and does not proceed or if pairing fails, try a factory reset.
-To factory reset, hold down the middle button on the bottom of the weather station, count to ~10 and release. The LED will start flashing very quickly as an indication that the factory reset has succeeded and then the station will restart.
 
-</span>
-</div>
```

Comment:
Please add this again but remove the instructions on how to do a factory reset. Just hyperlink in the text "try a factory reset" the new "Factory Reset" section you created below.